employees/views.py

```diff
@@ -55,6 +55,14 @@ def daily_hours_sum(queryset: QuerySet) -> dict:
     )
 
 
+def monthly_hours_sum(queryset: QuerySet) -> dict:
```

Comment:
Please also make this a `ReportManager` method, but I would like to understand it correctly - in this queryset you want to sum of all work hours for each author for each month? Because what it currently does is getting sum of all work hours for each author but from the whole queryset - if queryset contains data from more then a month, it is calculated for that whole time period, without dividing into months, only by author.

Reply:
@maciejSamerdak What about that ^ ?

Reply:
Correct, this issue would be resolved with next PRs, which add date filtering in list views showing only one month on page. Having that in mind I simplified this function, since the queryset provided in the view would already be filtered. Again, this feature is **fast-forwarded**, but @Karrp wanted it introduced ASAP, knowing this issue.